frontend/src/services/UserService.tsx

```diff
@@ -1,12 +1,15 @@
 import Keycloak from "keycloak-js";
 
+const AUTH_REALM = window.AUTH_REALM || process.env.AUTH_REALM;
```

Comment:
This would give you a boolean, you want to use a terenary operator here

Reply:
This seems to be working for me. It first reads from window.variable. If it is defined, it uses the defined value and if not it falls back to read from env variable.